lua/telescope/_extensions/fzf.lua

```diff
@@ -107,8 +107,8 @@ return require('telescope').register_extension {
     end
   end,
   exports = {
-    native_fzf_sorter = function()
-      return get_fzf_sorter()
+    native_fzf_sorter = function(custom_opts)
+      return get_fzf_sorter(custom_opts or opts)
```

Comment:
Good catch thanks :)
But opts should be undefined here. So just do `function(opts)`
 `return get_fzf_sorter(opts or { case_mode = "smart_case" })`

Reply:
wouldn't local override be useful instead of global default settings? I have a use case where I need to have fine-grained control over those settings. I import this sorter and use it in a customer finder.

Reply:
fixed